docs/src/test-global-setup-teardown-js.md

```diff
@@ -184,12 +184,21 @@ export default defineConfig({
 Start by creating a `global.setup.ts` file in the tests directory of your project. This will be used to seed the database with some data before all tests have run.
 
 ```js title="tests/global.setup.ts"
-// seed the database with some data
+import { test as setup } from '@playwright/test';
```

Comment:
I thought you would also change "Setup Example" to be bare-bones and link to auth doc for a login example?